server/datastore/mysql/hosts_test.go

```diff
@@ -1000,6 +1001,28 @@ func testHostsListMunkiIssueID(t *testing.T, ds *Datastore) {
 	require.ElementsMatch(t, []string{strings.Repeat("A", maxMunkiIssueNameLen), strings.Repeat("☺", maxMunkiIssueNameLen)}, names)
 }
 
+func testHostsListMissing(t *testing.T, ds *Datastore) {
+	for i := 0; i < 15; i++ {
+		_, err := ds.NewHost(context.Background(), &fleet.Host{
+			SeenTime:      time.Now().Add(-time.Duration(i) * time.Hour * 24),
+			OsqueryHostID: strconv.Itoa(i),
+			NodeKey:       fmt.Sprintf("%d", i),
+		})
+		require.NoError(t, err)
+		if err != nil {
+			return
+		}
+	}
+
+	filter := fleet.TeamFilter{User: test.UserAdmin}
+
+	hosts := listHostsCheckCount(t, ds, filter, fleet.HostListOptions{}, 15)
+	assert.Equal(t, 15, len(hosts))
+
+	hosts = listHostsCheckCount(t, ds, filter, fleet.HostListOptions{Missing10Days: true}, 4)
```

Comment:
nit: should we add a test for the explicit `false` too?

Reply:
Since false is the default value for bools in go, it would be identical to the test in 1019.